docker/docker-compose-init-db.yml

```diff
@@ -22,9 +23,6 @@ services:
       - type: bind
         source: "${ICUBAM_RESOURCES_PATH:-./resources}"
         target: /home/icubam/resources
-      - type: bind
-        source: "${ICUBAM_COMPOSE_CONTEXT:-.}"
-        target: /home/icubam/db
```

Comment:
SO is the DB now mounted as part of the resources folder?

Reply:
yes